boards/risc-v/esp32c3/esp32c3-devkit/Kconfig

```
@@ -65,6 +65,11 @@ choice
 
 endchoice
 
+config ESP32C3_SPIFLASH_FS_MOUNT_PTR
+	string "File-system Mount Pointer"
```

```suggestion
config ESP32C3_SPIFLASH_FS_MOUNT_PT
	string "File-system Mount Point"
```
nit: The correct expression would be "mount point"